examples/xusb.c

```diff
@@ -822,8 +822,9 @@ static int test_device(uint16_t vid, uint16_t pid)
 	int i, j, k, r;
 	int iface, nb_ifaces, first_iface = -1;
 	struct libusb_device_descriptor dev_desc;
-	const char* const speed_name[6] = { "Unknown", "1.5 Mbit/s (USB LowSpeed)", "12 Mbit/s (USB FullSpeed)",
-		"480 Mbit/s (USB HighSpeed)", "5000 Mbit/s (USB SuperSpeed)", "10000 Mbit/s (USB SuperSpeedPlus)" };
+	const char* const speed_name[7] = { "Unknown", "1.5 Mbit/s (USB LowSpeed)", "12 Mbit/s (USB FullSpeed)",
```

Comment:
I realize you've merely incremented this number, but better to remove it, and let the compiler do the counting.

then add another variable:

```c
size_t speed_count = sizeof(speed_name) / sizeof(char*);
```

and use it below instead of the magic 6.